app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +23,11 @@
             android:name="org.fossasia.openevent.general.AuthActivity"
             android:windowSoftInputMode="adjustPan">
         </activity>
-        <activity android:name=".auth.LoginActivity"></activity>
```

Comment:
Why is this removed?

Reply:
There doesn't exist any LoginActivity now

Reply:
Its auth activity

Reply:
Still, these changes not related to issue should be handled in separate PR. Take note from next time on